### DISCUSSION TITLE
Example Code for Predictive Maintenance Use Case

### DISCUSSION THREAD

**Discussion by yashveerdedha** [sebp/scikit-survival#419] [Q&A]
We are evaluating RandomSurvivalForest and CoxPHSurvivalAnalysis models for predictive maintenance use case , it would be a great help if anyone can share pointer for example code using "Scikit Survival".

**Comment by sebp** [comment]
Did you have a look at the user guides available at 
https://scikit-survival.readthedocs.io/en/latest/user_guide/index.html

**Reply by yashveerdedha** [reply]
Yes i've gone through the user guides still not found an complete example code for RandomSurvivalForest and CoxPHSurvivalAnalysis model.

**Comment by sebp** [comment]
Examples using CoxPHSurvivalAnalysis are at 
https://scikit-survival.readthedocs.io/en/latest/user_guide/00-introduction.html#Multivariate-Survival-Models

Example for RandomSurvivalForest are at 
https://scikit-survival.readthedocs.io/en/latest/user_guide/random-survival-forest.html
